ports/vulkan-memory-allocator-hpp/vcpkg.json

```
@@ -1,12 +1,12 @@
 {
   "name": "vulkan-memory-allocator-hpp",
-  "version": "3.0.1",
-  "port-version": 1,
-  "description": "C++ bindings for VulkanMemoryAllocator",
+  "version": "3.0.1-1",
```

I think this should be 
```suggestion
  "version": "3.0.1",
  "port-version": 2,
```

---

I don't think so. The version number of the upstream is being ported here is `3.0.1-1`. I think the `version` field should state the same. From my understanding the port-number should only increase if there's no changes to the upstream code, shouldn't it? @dan-shaw

---

Yes, you are correct. I think the version needs to be a `version-string` in that case: `version` should only accept dot separated sections

---

But `version-string` is not comparable with `version`. It is not an upgrade.

---

I think maybe we can use `version: 3.0.1.1` instead. `3.0.1 < 3.0.1.1` so it will work.

---

That sounds acceptable. I'll make the changes 👍